testing/scenario_app/ios/Scenarios/ScenariosUITests/LocalizationInitializationTest.m

```diff
@@ -0,0 +1,32 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "LocalizationInitialization.h"
+
+FLUTTER_ASSERT_ARC
+
+@implementation LocalizationInitializationTest
+
+- (void)setUp {
+  [super setUp];
+  self.continueAfterFailure = NO;
+
+  self.application = [[XCUIApplication alloc] init];
+  self.application.launchArguments = @[ @"--locale-initialization" ];
+  [self.application launch];
+}
+
+- (void)testNoLocalePrepend {
+  NSTimeInterval timeout = 10.0;
+
+  XCUIElement* textInputSemanticsObject =
+      [[[self.application textFields] matchingIdentifier:@"[en]"] element];
+
+  // The locales recieved by dart:ui are exposed onBeginFrame via semantics label.
+  // There should only be one locale, as we have removed the locale prepend on iOS.
+  XCTAssertTrue([textInputSemanticsObject waitForExistenceWithTimeout:timeout]);
+  XCTAssertEqualObjects([textInputSemanticsObject valueForKey:@"hasKeyboardFocus"], @(NO));
```

Comment:
I'm guessing `textInputSemanticsObject.hasKeyboardFocus` isn't an exposed property?

The better typed way to do this when you really have to (ignoring StackOverflow) is to put this at the top of your implementation file:
```objective-c
@interface XCUIElement (FlutterTests)
@property BOOL hasKeyboardFocus;
@end
...

Reply:
And then:
```objective-c
XCTAssertFalse(textInputSemanticsObject.hasKeyboardFocus);
```

Reply:
Ok, will keep in mind. I will actually remove this here as it is exercised in other tests.